class.jetpack.php

```diff
@@ -4429,7 +4443,8 @@ public function alert_auto_ssl_fail() {
 					$this.html( <?php echo json_encode( __( 'Checking', 'jetpack' ) ); ?> );
 					$( '#jetpack-recheck-ssl-output' ).html( '' );
 					e.preventDefault();
-					$.post( '/wp-json/jetpack/v4/recheck-ssl' )
+					var data = { action: 'jetpack-recheck-ssl', 'ajax-nonce': '<?php echo $ajax_nonce; ?>' };
```

Comment:
Yeah, that would be a good optimisation at some point in the future. TBH I imagine that once we go full-react all this stuff will have to be built/localised differently though.